tests/store/artifact/test_ftp_artifact_repo.py

```diff
@@ -66,6 +68,44 @@ def test_list_artifacts(ftp_mock):
     assert artifacts[1].file_size is None
 
 
+def test_list_artifacts_with_absolute_path(ftp_mock):
```

Comment:
Can we rename this test function to `test_list_artifacts_when_ftp_nlst_returns_absolute_paths` to clarify what this test does?